packages/react-celo/src/utils/network-watcher.ts

```diff
@@ -0,0 +1,20 @@
+import { ConnectorEvents } from '../connectors/common';
+import { Connector, Network } from '../types';
+import { getApplicationLogger } from './logger';
+
+function networkWatcher(connector: Connector, networks: Network[]) {
+  connector.on(ConnectorEvents.WALLET_CHAIN_CHANGED, (chainId) => {
```

Comment:
When starting on a network that isnt mainnet Valora (and any other wallet that only supports a single network) will automatically switch itself to the network it supports. I believe Valora never worked well with other networks if at all. and if  it technically worked before because we were not handling changing network from wallet properly. but since we do now and valora only supports mainnet this will break

Reply:
that's correct, Valora only supports mainnet AFAIK. To use Alfajores you need to have the valora-alfajores version installed, which is a different app altogether.